src/features/ISECompatibility.ts

```diff
@@ -0,0 +1,52 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+import * as vscode from "vscode";
+import { LanguageClient } from "vscode-languageclient";
+import { IFeature } from "../feature";
+
+/**
+ * A feature to implement commands to make code like the ISE and reset the settings.
+ */
+export class ISECompatibilityFeature implements IFeature {
+    private iseCommand: vscode.Disposable;
+    private defaultCommand: vscode.Disposable;
+    private settings = [
+        {section: "workbench.activityBar", setting: "visible", value: false},
+        {section: "debug", setting: "openDebug", value: "neverOpen"},
+        {section: "editor", setting: "tabCompletion", value: "on"},
+        {section: "powershell.integratedConsole", setting: "focusConsoleOnExecute", value: false},
+        {section: "powershell.integratedConsole", setting: "showOnStartup", value: false},
```

Comment:
Let's not do this one. I think we want the integrated console to show up

Reply:
Works for me. I found it in the docs and was kind of confused myself as I was pretty sure we ISE defaults to showing the terminal :man_shrugging: